app/src/main/java/de/tum/in/tumcampusapp/component/ui/cafeteria/CafeteriaMenuViewHolder.kt

```diff
@@ -14,7 +14,7 @@ class CafeteriaMenuViewHolder(itemView: View) : CardViewHolder(itemView) {
             cafeteriaNameTextView.text = cafeteria.name
             menuDateTextView.text = DateTimeFormat.mediumDate().print(cafeteria.nextMenuDate)
 
-            if (contentContainerLayout.childCount == 1) {
+            if (contentContainerLayout.childCount == 0) {
```

Comment:
:sparkles: magic numbers :sparkles:

Reply:
Added a clarifying comment

Reply:
Would love to see that Clarifying comment - where did you add it? :grinning:

Reply:
[here](https://github.com/TCA-Team/TumCampusApp/pull/1100/commits/8b3504a4f4dc46e8d4e52b3615119f665321d052)